jinahub/indexers/merger/SimpleMerger/tests/test_simple_merger.py

```diff
@@ -0,0 +1,34 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
```

Comment:
lets have a proper split of unit and integration tests

Reply:
good point, I added unit tests